.github/CONTRIBUTNG.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit helps, and credit will always be given.
+
+Please read the PAVICS [Developer Documentation](https://pavics-sdi.readthedocs.io/en/latest/dev/index.html) to get started.
+
+## Policy
+Since PAVICS is used in production by multiple organizations, this deployment repository also has a policy regarding contributions. 
+
+### Policy objectives:
+1. Keep production stable between upgrades
+2. Encourage PR to be merged more quickly
+3. Achieve the two previous objectives 
+      * by weighing down everyone's workflow as little as possible
+      * by having an approach that scales well as more nodes go into production and the number of PR increases
+
+### Policy rules:
+1. The repository has a main branch, "master", open to the community where contributions, "PR", are welcome. This master branch must not have owners and therefore no organization can block contributions to it.
```

Comment:
"must not have owners" -> is ownerless

Reply:
"must not" carries more meaning so we agreed to keep it